third_party/move/tools/move-formatter/tests/formatter/use/input1.move.fmt

```diff
@@ -0,0 +1,11 @@
+/// test_point: A list consisting of multiple items, with comments after the items
+
+module test_use {
+    use aptos_framework::aptos_coin::AptosCoin;
+    use aptos_framework::coin::{Self, Coin} /* use */ ;
```

Comment:
If you have coallescable use statements like:
```
    use aptos_framework::coin::A;
    use aptos_framework::coin::B;
```

will they get coallesced into:
```
    use aptos_framework::coin::{A, B};
```

In either case, add this into the test case to track it.

Reply:
we added a new test case:
tests/formatter/use/input6.move